README.md

```diff
@@ -77,7 +77,29 @@ All platforms use [TAO](https://github.com/tauri-apps/tao) to build the window,
 
 Tao uses [gtk-rs](https://gtk-rs.org/) and its related libraries for window creation and wry also needs [WebKitGTK](https://webkitgtk.org/) for WebView. So please make sure the following packages are installed:
 
-#### Arch Linux / Manjaro:
+#### Nix via NixOS
+Latest Packages for Development:
+```nix
+let
+  # Unstable Channel | Rolling Release
```

Comment:
is there a reason to recommend the unstable channel instead of stable?

Reply:
unstable channel on nix is essentially rolling release, thus in my subjective opinion, more suitable for development purposes.